warehouse-management-service/pkg/database/postgres/warehouse.go

```diff
@@ -9,7 +9,7 @@ import (
 	_ "github.com/lib/pq"
 )
 
-type queries interface {
+type warehouseQueries interface {
```

Comment:
Same for here. Should we add a go-mockgen command for this interface?

Reply:
Added